shared/src/userData.ts

```diff
@@ -29,7 +29,7 @@ export interface Business {
   readonly formationData: FormationData;
 }
 
-export const CURRENT_VERSION = 140;
+export const CURRENT_VERSION = 141;
```

Comment:
[sand]: Do we have a good enough reason to not just have this equal `Migrations.length`? I haven't been able to come up with one but curious on the thoughts of people who are more familiar with this code (everybody else)